src/smart-components/group/groups.js

```diff
@@ -37,10 +37,13 @@ const Groups = () => {
   const isAdmin = orgAdmin || userAccessAdministrator;
 
   const { groups, meta, filters, isLoading } = useSelector(
-    ({ groupReducer: { groups, isLoading, systemGroup } }) => ({
+    ({ groupReducer: { groups, isLoading, adminGroup, systemGroup } }) => ({
       groups: [
-        ...(systemGroup?.name?.match(new RegExp(groups.filters.name, 'i')) ? [systemGroup] : []),
-        ...(groups?.data?.filter(({ platform_default } = {}) => !platform_default) || []),
+        ...[
+          ...(adminGroup?.name?.match(new RegExp(groups.filters.name, 'i')) ? [adminGroup] : []),
+          ...(systemGroup?.name?.match(new RegExp(groups.filters.name, 'i')) ? [systemGroup] : []),
+        ],
```

Comment:
Do we need the first spread? I'd say we can omit it and have just the spread for `adminGroup` and `systemGroup`
```suggestion
        ...(adminGroup?.name?.match(new RegExp(groups.filters.name, 'i')) ? [adminGroup] : []),
        ...(systemGroup?.name?.match(new RegExp(groups.filters.name, 'i')) ? [systemGroup] : []),
```